src/librustc_resolve/macros.rs

```diff
@@ -151,7 +151,7 @@ impl<'a> base::Resolver for Resolver<'a> {
         invocation.expansion.set(visitor.legacy_scope);
     }
 
-    fn add_ext(&mut self, ident: ast::Ident, ext: Rc<SyntaxExtension>) {
+    fn add_ext(&mut self, ident: ast::Ident, kind: MacroKind, ext: Rc<SyntaxExtension>) {
```

Comment:
Since we have the `SyntaxExtension`, I don't think we need to add the `MacroKind` argument.